CFP.md

```diff
@@ -0,0 +1,68 @@
+CALL FOR PAPERS
+====
+
+--------------------------------------------------------------------------------
+*Workshop for NLP Open Source Software (NLP-OSS)*
+20 July 2018, Co-located with ACL 2018
+https://nlposs.github.io/
+
+Deadline for Long and Short Paper submission: 25th March 2018 
+(23:59, GMT-11)
+---------------------------------------------------------------------------------
+
+The First Workshop for NLP Open Source Software (NLP-OSS) will be co-located 
+with ACL 2018 at Melbourne, Australia on 20 July 2018. 
+
+Focusing more on the social and engineering aspect of NLP software 
+and less on scientific novelty or state-of-art models, the Workshop for NLP-OSS 
+is an academic forum to advance open source developments for NLP research, 
+teaching and application.
+
+NLP-OSS also provides an academic workshop to announce new software/features, 
+promote the collaborative culture and best practices that go beyond the conferences.
+
+We invite full papers (8 pages) or short papers (4 pages) on topics related to 
+NLP-OSS broadly categorized into (i) software development, (ii) scientific 
+contribution and (iii) NLP-OSS case studies.
+
+ - **Software Development**
+   - Designing and developing NLP-OSS
+   - Licensing issues in NLP-OSS
+   - Backwards compatibility and stale code in NLP-OSS
+   - Growing an NLP-OSS community
+   - Maintaining and motivating an NLP-OSS community
+   - Best practices for NLP-OSS documentation and testing
+   - Contribution to NLP-OSS without coding
+   - Incentivizing OSS contributions in NLP
+   - Commercialization and Intellectual Property of NLP-OSS
+   - Defining and managing NLP-OSS project scope
+   - Issues in API design for NLP
+   - NLP-OSS software interoperability
+   - Analysis of the NLP-OSS community
+
+ - **Scientific Contribution**
+   - Surveying OSS for specific NLP task(s)
+   - Demonstration and tutorial of NLP-OSS
+   - New NLP-OSS introductions
+   - Small but useful NLP-OSS
+   - NLP components in ML OSS
+   - Citations and references for NLP-OSS
+   - OSS vs experiment replicability
+   - Gaps between existing NLP-OSS
+   - Task-generic vs task-specific software
+
+ - **Case studies**
+   - Case studies of how a specific bug is fixed or feature is added
+   - Writing wrappers for other NLP-OSS
+   - Writing open-source APIs for open data
+   - Teaching NLP with OSS
```

Comment:
@alvations Would including "OSS in the industry" be desirable, if we were to pursue discussions for OSS both in academia and industry?

Reply:
Added it in https://github.com/nlposs/NLP-OSS/pull/17/commits/b4882dfed0a43830483caeccc80e0c347f06c93b

Reply:
Awesome!